packages/opencensus-core/test/test-console-exporter.ts

```diff
@@ -60,11 +44,15 @@ describe('NoopExporter', () => {
     it('should do anything', () => {
```

Comment:
`'should do nothing'` should be more accurate. After all, it's a no-op Exporter.

Reply:
Sure. Updated.